src/main/java/org/granitepowered/granite/GraniteStartupThread.java

```diff
@@ -87,11 +96,11 @@
     String[] args;
     BytecodeModifier modifier;
 
-    String serverVersion;
-    String apiVersion;
+    String serverVersion = "UNKNOWN";
+    String apiVersion = "UNKNOWN";
```

Comment:
Yes but for the times it does happen I'm sure you'd rather be prepared than not?